src/components/Header_type_1.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { routeTo } from 'redux-router-kit';
+import { getTranslator } from '../reducers/translation';
+import { RoundHeaderButton } from './Buttons';
+
+import styles from './Header_type_1.scss';
+import common from '../styles/common.scss';
+
+const HeaderType1Component = ({ i18n, targetTo }) =>
```

Comment:
Component name is not identical to file name. In this case the filename *needs* to be `HeaderType1.js`.

@Moejoe90 @banciur after you do a PR, you need to also go through the PR yourself and check for simple stuff like this.